guides/common/assembly_managing-content-views.adoc

```diff
@@ -1,52 +1,4 @@
-[id="Managing_Content_Views_{context}"]
-= Managing Content Views
-
-{ProjectNameX} uses Content Views to create customized repositories from the repositories.
-To do this, you must define which repositories to use and then apply certain filters to the content.
-These filters include both package filters, package group filters, errata filters, and module stream filters.
-You can use Content Views to define which software versions a particular environment uses.
-For example, a _Production_ environment might use a Content View containing older package versions, while a _Development_ environment might use a Content View containing newer package versions.
```

Comment:
Title and an introduction should stay in the assembly.
Mark -- see my comment for `assembly_synchronizing-content-between-foreman.adoc`